src/Command/CreateContentCommand.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Drupal\testsite_builder\Command;
+
+use Drupal\testsite_builder\ContentCreator;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Drupal\Console\Core\Command\ContainerAwareCommand;
+// phpcs:disable
+use Drupal\Console\Annotations\DrupalCommand;
+// phpcs:enable
+
+/**
+ * Class CreateContentCommand.
+ *
+ * @DrupalCommand (
+ *     extension="testsite_builder",
+ *     extensionType="module"
+ * )
+ */
+class CreateContentCommand extends ContainerAwareCommand {
+
+  /**
+   * The reporter instance.
+   *
+   * @var \Drupal\testsite_builder\ContentCreator
+   */
+  protected $contentCreator;
+
+  /**
+   * Constructs a new ContentCreatorCommand object.
+   */
+  public function __construct(ContentCreator $contentCreator) {
+    $this->contentCreator = $contentCreator;
+    parent::__construct();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function configure() {
+    $this
+      ->setName('testsite-builder:create-content')
+      ->addArgument('config', InputArgument::REQUIRED, $this->trans('commands.testsite_builder.create-content.arguments.config'))
+      ->addArgument('sampled-data', InputArgument::REQUIRED, $this->trans('commands.testsite_builder.create-content.arguments.sampled-data'))
+      ->addArgument('output-directory', InputArgument::REQUIRED, $this->trans('commands.testsite_builder.create-content.arguments.output-directory'))
+      ->setDescription($this->trans('commands.testsite_builder.create-content.description'));
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function execute(InputInterface $input, OutputInterface $output) {
```

Comment:
If we put the logic inside this execute() method into a service and the logic of the config-creators execute() method into another service, we could easily create a new command like called create:site, which does both tasks in one drupal console call, only requiring the report as a parameter.